src/types/index.ts

```diff
@@ -0,0 +1,22 @@
+export interface WatchtowerOptions {
```

Comment:
nit: consider using this `index.ts` to reexports all the types from model and any other file in `/types` dir. 

Also, consider moving all these types to `./cli.ts` or `./api.ts` or `./watchTower.ts`

Reply:
Tracking in #60.